integration-tests/src/test/resources/exporter/createWebhook.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upload
+monitoringExporterDir=$1
+domainNS=$3
+domainNS1=$4
+resourceExporterDir=$2
+monitoringExporterEndToEndDir=${monitoringExporterDir}/src/samples/kubernetes/end2end
+
+cd ${monitoringExporterEndToEndDir}
+docker build ./webhook -t webhook-log:1.0;
+if [ ${SHARED_CLUSTER} = "true" ]; then
+    docker login $REPO_REGISTRY -u $REPO_USERNAME -p $REPO_PASSWORD
+    echo "tag image " $REPO_REGISTRY/$REPO_USERNAME/webhook-log:1.0
+    docker tag webhook-log:1.0 $REPO_REGISTRY/weblogick8s/webhook-log:1.0
```

Comment:
reusing the same image name & tag over and over like this will lead to issues

Reply:
fixed to generate unique tag